awx/ui/client/src/notifications/edit/edit.controller.js

```diff
@@ -86,6 +86,10 @@ export default ['Rest', 'Wait',
                             }
                         }
                         else {
+                            if (data.notification_configuration.timeout === null ||
+                              !data.notification_configuration.timeout){
+                                $scope.timeout = 30;
```

Comment:
Nit: non blocker - can this be pulled from somewhere in the API side rather than being hard coded here ?